mycroft/audio/services/__init__.py

```diff
@@ -116,6 +116,27 @@ def restore_volume(self):
         Mycroft has lowered it using lower_volume().
         """
 
+    def get_track_length(self):
+        """
+        getting the duration of the audio in mlilliseconds
+        """
+
+    def get_track_position(self):
+        """
+        get current position in milliseconds
+
+          Args:
+                seconds (int): number of seconds of final position
+        """
+
+    def set_track_position(self, seconds=1):
+        """
+        go to position in seconds
+
+          Args:
+                seconds (int): number of seconds of final position
+        """
```

Comment:
Could you move the oneline description to the same line as the `"""` so it's consistent with the other methods in the class